js/ui/hierarchical_collection/ui.data_adapter.js

```diff
@@ -419,16 +421,28 @@ var DataAdapter = Class.inherit({
         return !this.options.searchValue.length || !!this._filterDataStructure(this.options.searchValue, [item]).length;
     },
 
+    _createCriteria: function(selector, value, op) {
```

Comment:
What is `op`? Parameter's name should be clear

Reply:
I think that this method is not needed. You can compile a getter function for searchExpr